bucket/onefetch.json

```diff
@@ -1,21 +1,20 @@
 {
-    "version": "2.2.0",
-    "description": "A command-line system information tool that displays information about your project directly on your terminal.",
+    "version": "2.5.0",
+    "description": "Git repository summary on your terminal.",
     "homepage": "https://github.com/o2sh/onefetch",
     "license": "MIT",
-    "notes": "You may have to enable ANSI Color Codes in your terminal. See https://stackoverflow.com/questions/51680709/",
```

Comment:
Why did you delete this?

Reply:
It's handled in the [code](https://github.com/o2sh/onefetch/blob/master/src/main.rs)

```rust
#[cfg(windows)]
let _ = ansi_term::enable_ansi_support();
```